### DISCUSSION TITLE
2.4 Launch Date

### DISCUSSION THREAD

**Discussion by ByteAndBits** [Security-Onion-Solutions/securityonion#10898] [2.4]
Hi,

do you know when SO 2.4 launches? We would implent SO in an enterprise network in the next months. So the new features of 2.4 are perfect for our enviroment. We don´t want to deploy 500 Wazuh agents and made a lot of config and 3 Month later the new Version of SO launches and we start again.

We don´t need the exact date, but a approximate release period will be great.

**Comment by dougburks** [comment]
2.4 RC2 is coming soon and 2.4 GA should come soon after that!

If you want to help out, you can try 2.4 RC1 and give us feedback:
https://blog.securityonion.net/2023/07/security-onion-24-release-candidate-1.html

**Reply by 8bitjoe** [reply]
Can we upgrade from RC releases to GA with soup?

**Reply by dougburks** [reply]
Yes, but please review the In-place Upgrades section of the new RC2 blog post for some important details:
https://blog.securityonion.net/2023/08/security-onion-24-release-candidate-2.html
